src/Microsoft.Health.Fhir.TemplateManagement.UnitTests/Utilities/StreamUtilityTests.cs

```diff
@@ -75,15 +76,24 @@ public void GivenACompressedTarGzFile_WhenDecompress_TheCorrectCountsOfFilesShou
         [MemberData(nameof(GetFilePathWithDigest))]
         public void GiveAFile_WhenCalculateDigest_ACorrectDigestShouldBeReturned(string filePath, string expectedDigest)
         {
-            var digest = StreamUtility.CalculateDigestFromSha256(File.ReadAllBytes(filePath));
+            var digest = StreamUtility.CalculateDigestFromSha256(Encoding.UTF8.GetBytes(File.ReadAllText(filePath).Replace("\r", string.Empty).Replace("\n", string.Empty)));
             Assert.Equal(expectedDigest, digest);
         }
 
         private void CompareTwoDictionary(Dictionary<string, byte[]> result, Dictionary<string, byte[]> expected)
         {
             foreach (var element in expected)
             {
-                Assert.Equal(result[element.Key], element.Value);
+                if (element.Value == null && result[element.Key] == null)
```

Comment:
This statement actually doesn't consider the condition `element.Value == null && result[element.Key] == null`, so is it possible to change it to

```csharp
if (element.Value != null || result[element.Key] != null)
{
    Assert.Equal(
        Encoding.UTF8.GetString(result[element.Key]).Replace("\r", string.Empty).Replace("\n", string.Empty),
        Encoding.UTF8.GetString(element.Value).Replace("\r", string.Empty).Replace("\n", string.Empty));
}
```